cves/kernel/CVE-2020-14386.yml

```diff
@@ -286,8 +315,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The vulnerability lied in the packet socket on the device driver layer, 
```

Comment:
Possibly change "lied" to "was found" or maybe even "lived"

Reply:
thank you!